README.md

```diff
@@ -1,83 +1,14 @@
-Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
+# Takeaway Challenge
 
- ```
+Develop a food ordering program in Ruby to fulfill the specific user stories, testimg with RSpec.
 
-Instructions
--------
+Click [here](https://github.com/amin-newcastle/takeaway-challenge/blob/2fc2cbba04b98697bc79fd071c572a1a54659654/domain_modelling_takeaway_challenge.pdf) to view my domain modeling pdf
```

Comment:
I really liked reading through this PDF. The user stories table and diagram was very clear.